src/components/biggive-text-input/biggive-text-input.scss

```diff
@@ -17,15 +15,20 @@
   }
   position: relative;
   .prompt {
-    position: absolute;
+    position: relative;
     z-index: 2;
     font-size: small;
-    top: -1em;
-    background-color: $colour-grey-background;
+    top: 1em;
     left: 4em;
-    padding-left: 1em;
-    padding-right: 1em;
     color: $colour-primary-blue;
+    max-width: calc(100% - 10em);
+    display: inline-block;
+
+    > span {
```

Comment:
PhpStorm is complaining about this span but I'm not sure why:

![image](https://github.com/thebiggive/components/assets/159481/6cc1cea8-dcca-48a8-a27a-83b2864f461f)